doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -240,30 +254,16 @@ libraries.
-| Imaging: Glance | https://docs.openstack.org/api-ref/image/v2/index.html#images  | Version 2.0 |
-| Block Storage: Cinder | https://docs.openstack.org/api-ref/block-storage/v3/index.html#api-versions  | Version 3.0 |
-| Object Storage: Swift | https://docs.openstack.org/api-ref/object-store/  | Version 1.0 |
-| Orchestration: Heat | https://docs.openstack.org/api-ref/orchestration/v1/index.html#api-versions  | Version 1.0 |
-| Acceleration: Cyborg | https://docs.openstack.org/api-ref/accelerator/v1/index.html  | Version 1.0 |
-
-**CLIs**
-
-| OpenStack Service | Link to Client | Client Minimum (Baseline) Version |
-|------------------|--------------------|------------------------|
-| Identity: Keystone | https://docs.openstack.org/python-keystoneclient/latest/using-api-v3.html | Version 3.10 |
-| Compute: Nova | https://docs.openstack.org/python-novaclient/latest/cli/index.html | Version 2.1 |
-| Networking: Neutron | https://docs.openstack.org/python-neutronclient/latest/cli/index.html | Version 2.0 |
-| Imaging: Glance | https://docs.openstack.org/python-glanceclient/latest/ | Version 2.0 |
-| Block Storage: Cinder | https://docs.openstack.org/python-cinderclient/latest/ | Version 3.0 |
-| Object Storage: Swift | https://docs.openstack.org/python-swiftclient/latest/ | Version 1.0 |
-| Orchestration: Heat | https://docs.openstack.org/python-heatclient/latest/ | Version 1.0 |
-| Acceleration: Cyborg | https://docs.openstack.org/python-cyborgclient/latest/ | Version 1.0 |
-| OpenStack Client (python) | https://docs.openstack.org/python-openstackclient/pike/index.html  | Version 3.2.1 |
+| OpenStack Service     | Link for API list                                    | **API Version** | **Minimal API Microversion** |
+|-----------------------|------------------------------------------------------|-----------------|------------------------------|
+| Identity: Keystone    | https://docs.openstack.org/api-ref/identity/v3/      | 3               | 3.8                          |
+| Compute: Nova         | https://docs.openstack.org/api-ref/compute/          | v2.1            | 2.53                         |
+| Networking: Neutron   | https://docs.openstack.org/api-ref/network/v2/       | v2.0            |                              |
+| Imaging: Glance       | https://docs.openstack.org/api-ref/image/v2/         | v2              | 2.5                          |
+| Block Storage: Cinder | https://docs.openstack.org/api-ref/block-storage/v3/ | v3              | 3.43                         |
+| Object Storage: Swift | https://docs.openstack.org/api-ref/object-store/     | v1              |                              |
+| Placement             | https://docs.openstack.org/api-ref/placement/        | v1              | 1.10                         |
+| Orchestration: Heat   | https://docs.openstack.org/api-ref/orchestration/v1/ | v1              |                              |
```

Comment:
Any particular reason to delete `cyborg` service?

Reply:
Simply because it's not listed in the upper-part.
Adding Cyborg as mandatory service is out of this change. Here it improves consistency between the 2 sections.

Reply:
We already have a PR #818 in progress, to add support. That PR will take care of adding it in above section. We can leave the unrelated data as it is because it will be coming in.
Removing it looks like it won't be used.

Reply:
The patch is about the actual content, nothing more.
It conforms with the collaborative rules at least as proposed by Github (we could have better handled patch dependencies with gerrit).

I don't see the point to block to block lots of (my?) PRs as done here.